lang/es_ES.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -3231,14 +3231,19 @@ msgid "You seem to be having trouble with your upgrade."
 msgstr "Parece que hay un problema con tu actualización."
 
 #: centreon-web/www/install/step_upgrade/step4.php:70
-msgid "You may refer to the line that causes problem in order to find out more about the issue."
+msgid "You may refer to the line in the specified file in order to correct the issue."
 msgstr "Puede consultar la línea que es problemática para obtener más información sobre el incidente."
 
 #: centreon-web/www/install/step_upgrade/step4.php:71
 #, php-format
 msgid "The SQL files are located in %s"
 msgstr "Los archivos SQL se encuentran en %s"
 
+#: centreon-web/www/install/step_upgrade/step4.php:87
+#, php-format
+msgid "Please check the \"upgrade.log\" and the \"sql-error.log\" located in \"%s\" for more details"
+msgstr ""
```

Comment:
if you put it empty, es translation will display empty.
If you cannot translate it, just remove those lines